src/main/resources/application.yml

```
@@ -16,4 +16,8 @@ spring:
         show_sql: true
         use_sql_comments: true
         highlight_sql: true
-    database-platform: org.hibernate.dialect.PostgreSQLDialect
\ No newline at end of file
+    database-platform: org.hibernate.dialect.PostgreSQLDialect
+
+jwt:
+  secret:
+    key: 6LcJCjCvxnX4tv6qji0bp6Aq7V9CMHeEIpTPOI693Oo=
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)